### PR TITLE
Feature/bump 2.4.0

### PR DESCRIPTION
**Pull request by CAMOBAP**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 2.4.0
+
+- Fix: call completionon WebContent process termination
```
Comment (e271828-) [inline CHANGELOG.md:3]:
```suggestion
- Fix: call completion on WebContent process termination
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+- Added: handle possible WebView navigation errors
```
Comment (e271828-) [inline CHANGELOG.md:4]:
```suggestion
- Added: handle possible WebView navigation errors
- Added: non-blocking initialization path
```

File: .github/workflows/workflow.yml [hunk 3]
```diff
@@ -125,7 +125,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        xcode: [ "${{ env.XCODE_VERSION }}" ]
+        xcode: [ '13.1' ]
```
Comment (e271828-) [inline .github/workflows/workflow.yml:128]: @CAMOBAP assume unintentional

Comment (CAMOBAP) [reply .github/workflows/workflow.yml:128]: Nope, unfortunately, `env` referencing doesn't work in the matrix. I tried but it doesn't work: https://github.com/hCaptcha/HCaptcha-ios-sdk/actions/runs/3986217990

**Review by e271828- — APPROVED** [review]

**Pull request merged**